app/views/reviews/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @review.title %></h1>
+
+<h2><%= @pizzeria.name %></h2>
+<h3><%= @pizzeria.street %><br/>
+  	<%= "#{@pizzeria.city}, #{@pizzeria.state} #{@pizzeria.zip_code}" %></h3>
```

Comment:
This might be a good place to add method to your model:

```ruby
class Pizzeria
  def address
    "#{city}, #{state} #{zip_code}"
  end
end
```

which you can then use in your views:

```HTML+ERB
<%= @pizzeria.address %>
```